### DISCUSSION TITLE
Which companies and projects are using nbb?

### DISCUSSION THREAD

**Discussion by borkdude** [babashka/nbb#91] [General]
Please share if your company or project is using nbb. You're welcome to provide a little bit of detail about how you are using it well. No strings attached.

This has several purposes:

- Your company might be mentioned in a sheet of a presentation about nbb or related projects. A link to a logo would be helpful.
- This issue would be a good way to notify you of any breaking changes in nbb (which we will try to prevent).
- Your company is listed on [this](https://github.com/borkdude/nbb/blob/main/doc/companies.md) page (TODO)
- It would help me a lot to understand who is using and how this project is being used.

If you would like to sponsor nbb or related projects, that'd be great:

https://github.com/sponsors/borkdude
https://opencollective.com/babashka

I'm also available for [commercial support](https://blog.michielborkent.nl/oss-highlights-nov-dec-2021.html#enterprise-support) if you need that.

Similar discussions for:

- [clj-kondo](https://github.com/clj-kondo/clj-kondo/discussions/1397)
- [babashka](https://github.com/babashka/babashka/discussions/1026)
- [SCI](https://github.com/babashka/sci/discussions/662)

**Comment by bobbicodes** [comment]
At Exercism, we were dealing with a problem of test runs on the Clojurescript track taking excessively long. I guess the nature of the issue is that each submission on the site is run cold, so compiling all the code to run our usual Clojurescript tests and display the results in a reasonable amount of time presented a challenge. One that we had already solved for Clojure using babashka, so thankfully, nbb came to our rescue here as well!

More details: https://github.com/exercism/clojurescript-test-runner/issues/3

**Reply by borkdude** [reply]
Thank you! That's really nice to hear.

**Reply by bobbicodes** [reply]
An interesting observation: it was written by Erik, who happens to be more fluent in bash than in Clojure, so he took a very different approach than I did by calling nbb from bash. I thought it was cool because it shows its utility beyond simply giving us Clojure-lovers a familiar environment.

**Reply by borkdude** [reply]
That's interesting! If he is fluent in Node.js / Javascript, he could also have done this in JS and then call the nbb API to evaluate CLJS.

**Reply by iHiD** [reply]
(cc @erikschierboom). 

@borkdude And thanks from me too :)

**Reply by ErikSchierboom** [reply]
Yeah thanks. I used bash because that was what I used to write similar code for other languages. I think it would be best if someone rewrote it at some point :)
